callme/proxy.py

```diff
@@ -123,6 +123,13 @@ def use_server(self, server_id=None, timeout=None):
             self._timeout = timeout
         return self
 
+    def destroy(self):
+        '''
```

Comment:
Please keep consistent style for docstrings ("""<docstring>""") with other parts in this module.